spyder/plugins/editor/lsp/manager.py

```diff
@@ -301,14 +301,16 @@ def update_python_config(self):
             'follow_imports': self.get_option('jedi_definition/follow_imports')
         }
 
+        # Advanced
+        advanced_external = self.get_option('advanced/external')
```

Comment:
Let's change this to

    external_server = self.get_option('advanced/external')

Reply:
You changed the names of other variables, but forgot to change precisely this name ;-)